.agents/skills/writing-pull-requests/SKILL.md

```
@@ -0,0 +1,25 @@
+# Writing Pull Requests for graz-sh/graz
+
+## Branch & Title
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)